.github/workflows/tests.yml

```diff
@@ -11,32 +11,48 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.6", "3.7", "3.8", "3.9"]
+    defaults:
+      run:
+        shell: bash
     env:
       OS: ${{ matrix.os }}
       PYTHON: ${{ matrix.python-version }}
     name: Python ${{ matrix.python-version }} on OS ${{ matrix.os }}
     steps:
     - name: Acquire sources
-      uses: actions/checkout@v2
+      uses: actions/checkout@v2.3.4
```

Comment:
This is not needed. When using `@v2`, it will automatically use the most recent release of the v2 series.